src/main/java/com/google/cloud/genomics/utils/Paginator.java

```diff
@@ -203,9 +204,9 @@ private Datasets(Genomics genomics) {
       super(genomics);
     }
 
-    @Override Genomics.Datasets.List createSearch(Genomics.Datasets api, Long request,
+    @Override Genomics.Datasets.List createSearch(Genomics.Datasets api, String request,
```

Comment:
The method signature needs to include a RequestT parameter per https://github.com/deflaux/utils-java/blob/master/src/main/java/com/google/cloud/genomics/utils/Paginator.java#L996

@melissachang Is that String value really unused? https://developers.google.com/resources/api-libraries/documentation/genomics/v1/java/latest/com/google/api/services/genomics/Genomics.Operations.List.html says "Lists operations that match the specified filter in the request."